backup_script.py

```diff
@@ -26,7 +114,34 @@ def load_api_key():
 
     return key
 
+def load_data(api_key):
+    data = {'members': get_all_members(api_key)}
+    return json.dumps(data)
```

Comment:
Include these parameters for nicer file formatting. 
```suggestion
    return json.dumps(data, ensure_ascii=False, indent=4)
```